src/game/enemies/marun.c

```diff
@@ -170,9 +171,9 @@ void Task_8063858(void)
         sprite->anim = gUnknown_080D210C[2].anim;
         sprite->variant = gUnknown_080D210C[2].variant;
         sprite->prevVariant = 0xFF;
-        sprite->frameFlags |= 0x6A;
+        sprite->frameFlags |= SPRITE_FLAG_SHIFT_106;
```

Comment:
frameFlags consist of multiple bits.
That's why `SPRITE_FLAG_SHIFT_106` doesn't make any sense, with frameFlags is only 32bits.

Use `(SPRITE_FLAG(blah_x) | SPRITE_FLAG(blah_y) | etc.)`.